filter_plugins/ceph-ansible.py

```diff
@@ -0,0 +1,13 @@
+from ansible import errors
+
+def only_one_true(a, *args):
+    params = list(args)
+    params.append(a)
```

Comment:
commented above. there are a few ways to do this and it all involves the syntax we use with the filter outside of this file